frontend/src/app/AppInsights.js

```diff
@@ -0,0 +1,20 @@
+// AppInsights.js
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { createBrowserHistory } from "history";
+
+const browserHistory = createBrowserHistory({ basename: "" });
+const reactPlugin = new ReactPlugin();
+const appInsights = new ApplicationInsights({
+  config: {
```

Comment:
Is there a way to pass along any additional tags with the log messages? Currently, we're using a global app analytics workspace, so it would be nice to be able to segment logs based on whether they're coming from dev/test

Reply:
i think we can probably add any other key:value pair in our tracking requests, and then filter by that.

What about having a dedicated dev/test analytics instance (like the one this PR is currently hooked up with)? We can point to it in our dev and stage environments to not clutter our production instance with a bunch of useless logs.

Reply:
In general, I like to have all the logging and messaging centralized into a single location, but happy to change course. Thoughts @pete-usds @emmanuel-apau  @benwarfield-usds

Reply:
I think separating production-equivalent from non-prod is preferable. For one thing, if we have a "PII spill" in non-prod, I don't want auditors coming after us and making us prove that it wasn't real PII. 😁